package.json

```diff
@@ -23,11 +23,12 @@
   },
   "scripts": {
     "build": "gulp",
-    "pretest": "npm run lint",
-    "test": "gulp test",
-    "html-report": "nyc report --temp-directory=./coverage --reporter=html",
-    "jest": "jest --coverage",
-    "coveralls": "nyc report --temp-directory=./coverage --reporter=text-lcov | coveralls",
+    "pretestonly": "chromedriver &",
```

Comment:
I tried. Due to some reason it didn't work out. It doesn't run our JS code for some reason. Debugging that is hard.

Anyway, chromedriver works pretty well and it's pretty fast.
(BTW: chromedriver is installed via NPM, so no need to setup anything)